tools/piegen/piegen.js

```diff
@@ -23,21 +23,75 @@ function getRandomNumber(min, max) {
   return Math.max(min, Math.round((Math.random() * max)));
 }
 
-const MIN_WEIGHT = 300; // In grams.
-const MAX_WEIGHT = 1000; // In grams.
-const MAX_PRICE = 800;
+const ADJACTIVES = [
```

Comment:
Global replace 'ADJACTIVES' with 'ADJECTIVES'.

Reply:
It did look suspicious to me;)